src/main/proto/spine/text/text.proto

```diff
@@ -43,15 +43,39 @@ message Text {
 // A position in the text.
 message Position {
 
-    // A zero-based index of the line of text.
-    //
-    // A value of -1 means "Not found".
-    //
-    int32 line = 1 [(min).value = "-1"];
+    oneof marker {
+
+        // A zero-based index of the line of text.
+        //
+        // A value of -1 means "Not found".
+        //
+        int32 line = 1 [(min).value = "-1"];
+
+        // A position beyond the existing text.
+        PositionBeyondText beyond_text = 3;
+    }
 
     // A zero-based position inside the [line].
     //
     // If the [line] property equals -1 the value of the column field has no meaning.
     //
+    // If the line marker is set beyond the existing text, user may choose to either:
```

Comment:
Can we have a link from `beyond` or `beyond the existing text` to the existing `beyond_text` field, please? People usually learn from repetitions better.